index.html

```diff
@@ -389,9 +389,25 @@ <h2 class="section-heading text-uppercase">Culture: what's it like working at Da
           <blockquote class="blockquote padding">
             <div class="blockquote-content padding">
               <p class="oblique">
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<br/>
-                Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.<br/>
-                Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                "DashboardHub has adopted a startup culture that promotes inclusivity and flexibility for those involved. A key aspect in making this a successful culture is the adoption of Open Source development in harmony with Remote Working. All of the team at DashboardHub are treated as equals and trusted to perform their tasks autonomously knowing they have the full support of the company's founders." <footer>- Andrew Cunliffe, Consultant</footer>
+              </p>
+            </div>
+          </blockquote>
+        </div>
+        <div class="row">
+          <blockquote-inverse class="blockquote padding">
+            <div class="blockquote-content padding">
+              <p class="oblique">
+                "Transparency, trust and inclusion are all vital to our team’s success." <footer-inverse>- Eddie Jaoude, CEO</footer-inverse>
```

Comment:
I think the `footer` should not be inside the `p` tag and needs the class `blockquote-footer`